docs/working-with-anp.md

```diff
@@ -0,0 +1,295 @@
+# Deploy apiserver-network-proxy(anp)
+
+## Purpose
+
+For the member cluster that joins karmada in pull mode, we need to provide a method to connect the network between the karmada control plane and the member clusters, so that karmada-aggregated-apiserver can access the pull mode member clusters.
```

Comment:
```suggestion
For a member cluster that joins karmada in pull mode, we need to provide a method to connect the network between the karmada control plane and the member cluster, so that karmada-aggregated-apiserver can access this member cluster.
```